tests/test_events/test_query_all_events.py

```diff
@@ -11,7 +11,7 @@ def test_query_events(self):
         """
         Test a user can query for all events
         """
-        CommonTestCases.user_token_assert_equal(
+        CommonTestCases.admin_token_assert_equal(
```

Comment:
@Bonifase I suggest  the docstring should be updated to something close  to `Test an admin can query for all events in the location`

Reply:
Any user can query all events, the reason admin token is used is that the regular user token does not have location attribute in it in the test environment

Reply:
Got it